lib/pavilion/commands/config.py

```diff
@@ -244,7 +244,8 @@ def create_config_dir(self, pav_cfg: config.PavConfig, path: Path,
             raise ConfigCmdError("Error writing config file at '{}'"
                                  .format(config_file_path), err)
 
-        for subdir in 'hosts', 'modes', 'tests', 'os', 'test_src', 'plugins', 'collections':
+        for subdir in ('hosts', 'modes', 'tests', 'os', 'test_src',
+            'plugins', 'collections', 'suites'):
```

Comment:
Since we're replacing test_src and tests with suites, let's get rid of those here.

Reply:
Since we're technically deprecating those, would it make sense to keep them around for awhile for backwards compatibility? Otherwise we'll have to restructure all the existing tests as soon as this is rolled out.